servant/src/Servant/API/UVerb.hs

```diff
@@ -70,19 +69,14 @@ instance (HasStatus a, HasStatuses as) => HasStatuses (a ': as) where
   type Statuses (a ': as) = StatusOf a ': Statuses as
   statuses _ = statusOf (Proxy :: Proxy a) : statuses (Proxy :: Proxy as)
 
+-- | A simple newtype wrapper that pairs a type with its status code.  It
+-- implements all the content types that Servant ships with by default. If
+-- you're using Servant with a content type that is not in the core servant
+-- packages, and you want to avoid orphan instances you can very easily define your own newtype instead.
+--
+-- newtype MyWithStatus (k :: Nat) a = WithStatus a deriving (ToMyContent, FromMyContent)
 newtype WithStatus (k :: Nat) a = WithStatus a
-  deriving (Eq, Show, GHC.Generic)
-
-instance (GHC.Generic (WithStatus n a), ToJSON a) => ToJSON (WithStatus n a)
-
-instance (GHC.Generic (WithStatus n a), FromJSON a) => FromJSON (WithStatus n a)
-
-instance MimeRender ctype a => MimeRender ctype (WithStatus _status a) where
-  mimeRender contentTypeProxy (WithStatus a) = mimeRender contentTypeProxy a
-
-instance MimeUnrender ctype a => MimeUnrender ctype (WithStatus _status a) where
-  mimeUnrender contentTypeProxy input =
-    WithStatus <$> mimeUnrender contentTypeProxy input
```

Comment:
why did you remove these?

Reply:
Read the commit message and read the issue this PR is trying to fix.

Reply:
(Long story short. These instances already existed; and this instance you defined here was overlapping; which meant code using WithStatus could never compile; but it wasn't caught by the test-suite as we didn't run `haskell-ci regenerate` before merging the uverb PR)

Reply:
> why did you remove these?

sorry, found the answer in the commit message.  
38db23b

Reply:
Hmmmm I just came up with a simpler solution..